etc/pip_install/toree/toreeapp.py

```diff
@@ -18,38 +18,25 @@
 import os.path
 import sys
 import json
-from traitlets import Unicode, Dict
+import shutil
+from traitlets import Unicode, Dict, Set
 from jupyter_client.kernelspecapp  import InstallKernelSpec
 from jupyter_core.application import base_aliases
 from traitlets.config.application import Application
 from toree._version import __version__, __commit__
+from jupyter_client.kernelspec import KernelSpec
 
-KERNEL_SPEC= {
-    'display_name': '',
-    'language_info': { 'name': 'scala' },
-    'argv': [
-        '',
-        '--profile',
-        '{connection_file}'
-    ],
-    'codemirror_mode': 'scala',
-    'env': {
-        'SPARK_OPTS': '',
```

Comment:
I was wondering if in this file we could do
```
'SPARK_OPTS': ''<what we get through the install script>:${SPARK_OPTS}"
```

That way users can add or replace settings through the env variable.